styleguide/components/41-organisms/feedback-form/_feedback-form.scss

```
@@ -21,5 +21,9 @@
     textarea {
       min-height: 5.5rem;
     }
+
+    .form-item-column {
+      flex: 100% 0 0;
```

CSS `flex` property can have 3 values: `flex-grow | flex-shrink | flex-basis` according to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex#Syntax). 

That means that flex-grow has a value of `100%` which is invalid (also see [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow#Values)).

```suggestion
      flex: 0 0 100%;
```

---

Thanks
Strangely, firefox breaks it down correctly to flex-basis 100%.
It's magical!